components/CollectiveNavbar.js

```diff
@@ -319,17 +359,57 @@ const CollectiveNavbar = ({
               )}
             </div>
           )}
-          {/* CTAs for v2 navbar */}
+          {/* CTAs for v2 navbar & admin panel */}
           {parseToBoolean(getEnvVar('NEW_COLLECTIVE_NAVBAR')) && (
-            <Fragment>
+            <Container
+              display={['none', null, 'flex']}
+              flexDirection={['column', null, 'row']}
+              flex="1 1 fit-content"
+              backgroundColor="#fff"
+              zIndex={1}
+            >
+              {isAdmin && (
+                <Flex mx={2} alignItems="center">
+                  <Link
+                    route={isEvent ? 'editEvent' : 'editCollective'}
+                    params={
+                      isEvent
+                        ? { parentCollectiveSlug: collective.parentCollective?.slug, eventSlug: collective.slug }
+                        : { slug: collective.slug }
+                    }
+                  >
+                    <Container
+                      display="flex"
+                      flexGrow={1}
+                      alignItems="center"
+                      borderRadius={8}
+                      background="rgba(72, 95, 211, 0.1)"
+                      px={3}
+                      py={1}
+                    >
+                      <BlueSettingsIcon size={20} />
```

Comment:
:ice_cream: Icons from `styled-icons` support the `color` prop. You can directly write:

```suggestion
                      <Settings size={20} color="rgb(48, 76, 220)" />
```

Reply:
I've tried that before and it doesn't work... although I think I might have been using the theme colors like "blue.700". I will try this.

Reply:
Yep, theme colors indeed don't work because they aren't wrapped with `styled-system`. We could introduce a helper for that in the future